agent/attacher/src/jp/co/ntt/oss/heapstats/attacher/AgentAttacher.java

```diff
@@ -58,6 +58,23 @@ public static void attachProcess(VirtualMachineDescriptor vmd,
                                         vmd.displayName());
       }
 
+      /* Recommend jcmd instead of Attach API if jdk9 or later */
+      /* See https://bugs.openjdk.java.net/browse/JDK-8177154 */
+      String[] versions = vm.getSystemProperties().getProperty("java.version").split("\\.", 3);
+      /* Check early access. See https://bugs.openjdk.java.net/browse/JDK-8061493 */
```

Comment:
Your comment makes sense.
However, I think that we should change [existing codes](https://github.com/HeapStats/heapstats/blob/788c84f7351e473464897b7bfcee1edea6e1e6df/agent/src/heapstats-engines/jvmInfo.cpp#L527) as well. But this PR mainly aims to respond [JDK-8177154](https://bugs.openjdk.java.net/browse/JDK-8177154) as described before. So I want to change existing codes and response your comment by new PR if you agree.